themes/xresources/theme.lua

```diff
@@ -64,21 +65,11 @@ theme = theme_assets.recolor_layout(theme, theme.fg_normal)
 
 -- Recolor titlebar icons:
 --
-local function darker(color_value, darker_n)
-    local result = "#"
-    for s in color_value:gmatch("[a-fA-F0-9][a-fA-F0-9]") do
-        local bg_numeric_value = tonumber("0x"..s) - darker_n
-        if bg_numeric_value < 0 then bg_numeric_value = 0 end
-        if bg_numeric_value > 255 then bg_numeric_value = 255 end
-        result = result .. string.format("%2.2x", bg_numeric_value)
-    end
-    return result
-end
 theme = theme_assets.recolor_titlebar(
     theme, theme.fg_normal, "normal"
 )
 theme = theme_assets.recolor_titlebar(
-    theme, darker(theme.fg_normal, -60), "normal", "hover"
+    theme, gcolor.darken(theme.fg_normal, 0.6), "normal", "hover"
```

Comment:
here and next i see you swapped direction from making lighter color into making darker one